### PR TITLE
removes backup locations for cv and introduce properties

### PR DESCRIPTION
**Pull request by rizwan-ishtiaq**

### CODE REVIEW

**Comment by jluo321** [comment]
in CVSystemProperties, only getDatabaseTypesLocation method throw exception. This method may not need to throw exception (based on your call of this method)

**Comment by jluo321** [comment]
in the CV, there are different ways to access data, some call repo, some call service. Can you make it consistent

File: controlled-vocabulary/src/test/java/org/uniprot/cv/evidence/GoEvidencesPropertyLocationTest.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Optional;
+
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+import org.uniprot.cv.Reloader;
+
+@TestInstance(Lifecycle.PER_CLASS)
+class GoEvidencesPropertyLocationTest {
```
Comment (eddturner) [inline controlled-vocabulary/src/test/java/org/uniprot/cv/evidence/GoEvidencesPropertyLocationTest.java]: Should all `Go...` names -> `GO...`? It stands for Gene Ontology, so should be GO like we did in QuickGO?

(sorry this comment is a bit off topic!)

File: controlled-vocabulary/src/test/java/org/uniprot/cv/evidence/GoEvidencesPropertyLocationTest.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        writeDataInfile();
+
+        System.setProperty(GAF_ECO_LOCATION, tmpFile.getAbsolutePath());
+
+        /*
+         * singleton enum pattern, jvm will load it only once depending on property set or not
+         * need to load it again (to test different behaviours) with different class loader
+         */
+        Class<?> goEvidences =
+                Class.forName("org.uniprot.cv.evidence.GoEvidences", true, new Reloader());
```
Comment (eddturner) [inline controlled-vocabulary/src/test/java/org/uniprot/cv/evidence/GoEvidencesPropertyLocationTest.java]: GoEvidences -> GOEvidences, etc.

**Pull request merged**